iroha/src/sumeragi.rs

```diff
@@ -32,6 +36,8 @@ pub enum Message {
     Signed(SignedBlock),
     /// Is sent by proxy tail to validating peers and to leader, when the block is committed.
     Committed(SignedBlock),
+    /// Is sent when the node votes to change view due to commit timeout.
```

Comment:
What view?

Reply:
This is from the BFT literature where a "view change" is changing the leader

Reply:
We do not have an entity called view except world state view in Iroha now. Should we stick to this literature term?